### Задачи 1 и 2

###### Контрольная работа состоит из пяти вопросов. На каждый вопрос приведено четыре варианта ответа, один из которых правильный. Случайная величина X задаёт число правильных ответов при простом угадывании. Найдите математическое ожидание M(X) данной случайной величины. Найдите дисперсию D(X) случайной величины X.

Вероятность правильного ответа на один вопрос $p^+=1/4$, вероятность неверного ответа $p^-=3/4$, значит для X=0 нужно ответить неправильно 5 раз подряд, вероятность такого события $P_{x=0}=(p^-)^5=243/1024$ а число способов так ответить на 5 вопросов $С_5^0=1$

Аналогично для X=1: Считаем, что правильный ответ один ($p^+=1/4$), а остальные четыре неверные (($p_4^-=(3/4)^4$)). Число способов правильно ответить на один вопрос из 5 - $С_5^1=5$, каждый из них реализуется с вероятностью $p_1^+=p^+p_4^-$. Эти реализации независимые, значит вероятности складываются, и вероятность того, что X=1:
$$P_{x=1}=С_5^1 \cdot p^+ \cdot p_4^- = 5\cdot1/4\cdot(3/4)^4=405 / 1024$$
Аналогично для X=2:
$$P_{x=2}=С_5^2 \cdot p^+_2 \cdot p_3^- = 10 \cdot(1/4)^2 \cdot (3/4)^3 = 270 / 1024$$
для X=3:
$$P_{x=3}=С_5^3 \cdot p^+_3 \cdot p_2^- = 10 \cdot(1/4)^3 \cdot (3/4)^2 = 90 / 1024$$
для X=4:
$$P_{x=4}=С_5^4 \cdot p^+_4 \cdot p_1^- = 5 \cdot(1/4)^4 \cdot 3/4 = 15 / 1024$$
для X=5:
$$P_{x=5}=С_5^5 \cdot p^+_5 \cdot p_0^- = 1 \cdot(1/4)^5 = 1 / 1024$$
Убеждаемся, что $$P_{x=0}+P_{x=1}+P_{x=2}+P_{x=3}+P_{x=4}+P_{x=5}=1$$

Запишем распределение случайной величины и вычислим M(X) и D(X):
<table>
<tr>
    <td> Величина</td>
    <td> X=0</td>
    <td> X=1</td>
    <td> X=2</td>
    <td> X=3</td>
    <td> X=4</td>
    <td> X=5</td>
</tr>
<tr>
    <td> Вероятность $P_X$</td>
    <td> 243/<br>1024</td>
    <td> 405/<br>1024</td>
    <td> 270/<br>1024</td>
    <td> 90/<br>1024</td>
    <td> 15/<br>1024</td>
    <td> 1/<br>1024</td>
</tr>
</table>

$$M(X)= 0 \cdot P_{x=0}+1 \cdot P_{x=1}+2 \cdot P_{x=2}+ 3 \cdot P_{x=3}+ 4 \cdot P_{x=4}+5 \cdot P_{x=5}=1,25$$


In [2]:
m = 1.25
x_probabilities = [243/1024, 405/1024, 270/1024, 90/1024, 15/1024, 1/1024]
x_values = [0, 1, 2, 3, 4, 5]
Disp = 0
tmp = 0
for x in x_values:
    Disp=Disp+(x - m)**2*x_probabilities[tmp]
    tmp += 1
print(f"Дисперсия D(X)={Disp}")

Дисперсия D(X)=0.9375


#### Ответ: &nbsp;&nbsp;&nbsp;     M(X)=1.25   &nbsp;&nbsp;&nbsp;   D(X)=0.9375

### Задача 3

###### Пользователь получает в среднем 10 писем со спамом на свой почтовый ящик за сутки. Найти число N, такое, что с вероятностью 0.95 пользователь получит не более N писем со спамом за текущий день.

Вероятность получения $k\leqslant N$ писем задается с помощью распределения Пуассона:
$$0.95=\sum\limits_{k=0}^N\dfrac{10^k e^{-10}}{k!}$$
Решение этого уравнения относительно N можно найти численно (подбором), например с помощью desmos.

#### Ответ: N=15


### Задача 4

###### Производятся выстрелы по мишени. Вероятность попадания в мишень при одном выстреле равна 0.01. Сколько выстрелов нужно сделать чтобы быть уверенным с вероятностью 0.9, что хотя бы 6 раз будет совершено попадание?

по условию $P(X\geqslant6)=0.9$, значит $P(X\leqslant5)=1 - P(X\geqslant6) = 0.1$

Вероятность попасть в мишень не больше 5-ти раз, сделав n выстрелов задается суммой биномиальных распределений:
$$P(X\leqslant5)=\sum\limits_{k=0}^5C_k^np^k(1-p)^{(n-k)}=0.1$$
Но решение этого уравнения "в лоб" или с помощью неполной бета-функции, представляется нерациональным.
Считая, что искомое n достаточно велико, построим приближенное решение, используя предельный случай бинамального распределения - распределение Пуассона с параметром $\lambda=np=0.01n$:
$$P(X \leqslant 5) = \sum\limits_{k=0}^5\dfrac{0.01^kn^k e^{-0.01n}}{k!}=0.1$$

Это уравнение уже можно быстро решить относительно n. Например, пакет wolframalpha позволяет это сделать.

Получается примерное значение n = 927
#### Ответ: для 6 и более попаданий в мишень (с вероятностью 0.9) требуется примерно 927 выстрелов.
